ui/src/criteria/concept.test.tsx

```diff
@@ -138,23 +138,32 @@ beforeAll(() => {
     insertGroup(
       action.payload.id,
       GroupKind.Included,
-      createCriteria({
```

Comment:
As above, fixing formatting issues is good though it shouldn't be in the same PR as other changes. However, in this case I don't understand why it changed, since `npx prettier --write .` doesn't reformat it for me. Do you have something else reformatting files with different options?

Reply:
I think this is not from `npx prettier --write .`  it caused by changing the data structure of createCriteria from L141-L147

Reply:
Oh, you're right. That makes sense.